cmd/syncer/helm/converter.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v3"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8syaml "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	"k8s.io/klog/v2"
+	sigyaml "sigs.k8s.io/yaml"
+
+	edgev1alpha1 "github.com/kcp-dev/edge-mc/pkg/syncer/apis/edge/v1alpha1"
+)
+
```

Comment:
No, there isn't. I will fill the command description and update README in this PR. This command is not related with Syncer but just an utility to generate EdgeSyncConfig manifest from yaml manifests returned by Helm (`helm template kyverno --dry-run`). I would like to leave this command.

Reply:
FYI, two smaller PRs are easier to review than one bigger PR.

Reply:
Yes, I totally agree with that. I will do that from the next.

Reply:
Today I see no reference to this command.

```console
(venv) mspreitz@mjs12 kubestellar % find * \( -path docs/venv -prune \) -or \( -name "*.go" -exec grep helm \{\} \; -print -exec echo \; \)
	fs := pflag.NewFlagSet("helmconverter", pflag.ExitOnError)
	fs.StringVar(&pathToHelmTemplate, "path-to-helm-template", "", "path to Helm tamplate file")
cmd/syncer/helm/converter.go

(venv) mspreitz@mjs12 kubestellar % 
```

It is a problem to build a variant executable named `helm` and put it in the KubeStellar `bin` directory because that will shadow the regular `helm` for users of KubeStellar.

Reply:
Today I can not find any documentation of the use of this command.